### DISCUSSION TITLE
SEB Reset Utility auto run?

### DISCUSSION THREAD

**Discussion by bonbon-2** [SafeExamBrowser/seb-win-refactoring#249] [Q&A]
Hello,

Is it possible to run the SEB Reset Utility automatically with default choices.?
So that users only have to double-click on the icon, and don't have to choose, Reset system configuration to default values and then enter a username, to start the Reset Utility and then exit it?
I ask this because very often we can no longer shut down Windows after taking an exam. The default settings are not released.
To avoid having to tell 20 people what to do at that moment (double click, choose this, type username, exit program, restart Windows) it would be very easy if they only have to double click on the icon SEB reset utility.

Hope someone can help me with this.

**Comment by dbuechel** [comment]
Doing this automatically when starting the utility is not possible, unfortunately, as the user needs to specify the user name under which they executed SEB. It furthermore could make undesired changes when a user doesn't want to reset all values but only those specific ones which were changed when executing SEB.

Given your information, I'd rather attempt to investigate the root cause of your problem: Are there any errors or warnings in the service logs of SEB (to be found under `%LocalAppData%\SafeExamBrowser\Logs`)?

**Reply by bonbon-2** [reply]
I know what the problem is, last april I discussed this with you.

Windows update and the fact that this computers are only on for taking an axam. Perhaps ones a week 1,5 hour. The rest of the time the laptops are in a locked case.

The issue is:
instead of shutdown/sleep/logoff we see:"There are currently no power options available"
Ctrl+Alt+Delete no taskmanager

After running SEB Reset Utility, and restart Windows it's possible to shut down the laptop.

So I hoped I could solve this problem with  autorun SEB Reset Utility

**Reply by dbuechel** [reply]
Which Windows and SEB versions are you using?

**Reply by dbuechel** [reply]
You might be able to automate your use case with a third-party tool which automatically navigates the UI of the reset utility.

**Reply by bonbon-2** [reply]
Windows 10 with SEB version 3.3.1.388

**Reply by MIAkhan2222** [reply]
i have a exam on 15 june.please tell me to can i solve this problem.I Don't know what to do about this problem SEB reset utility

**Reply by dbuechel** [reply]
@MIAkhan2222  You'd need to be a bit more specific: What exactly is your problem resp. what isn't working the way you'd expect it to work?
